README.md

```diff
@@ -61,6 +61,59 @@ If you have a preferred import path to suggest, append the suggestion after a `=
 -paths "fmt.{Errorf}=github.com/pkg/errors.{Errorf}"
 ```
 
+### Ignoring problems
+
+If you want to ignore a problem reported by `faillint` you can add a linter directive.
+The format of a directive is based on the format other linters, such as `staticcheck`, use and there are two types: line-based and file-based.
+
+#### Line-based linter directives
+
+Line-based linter directives can be applied to imports or functions you want to tolerate.  The format is,
+
+```go
+//faillint:ignore reason
+```
+
+For example,
+
+```go
+package a
+
+import (
+        //faillint:ignore Whatever your reason is.
+        "errors"
+        "fmt" //faillint:ignore Whatever your reason is.
+)
+
+func foo() error {
+        //faillint:ignore Whatever your reason is.
+        return errors.New("bar!")
+}
+```
+
+#### File-based linter directives
+
+File-based linter directives can be applied to ignore `faillint` problems in a whole file.  The format is,
```

Comment:
Can you add a comment that the ignore directive has to be before the `package` keyword? Or is there any other case that would also work?

Reply:
Added clarification; in 39860172ce6b9bf105285ce19ff50cab88be6a8f.  If you want it anywhere in the file that can be done but it's probably better to keep it limited to the package docs.